gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-notifications.spec.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER, API_PUBLISHER_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { ApiImport } from '@model/api-imports';
+import apiDetails from 'ui-test/support/PageObjects/Apis/ApiDetails';
+import { PlanStatus } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import { PlansFaker } from '@gravitee/fixtures/management/PlansFaker';
+
+let api: ApiImport;
+
+describe('Notifications page', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit(`/#!/environments/default/apis/${api.id}/notification-settings`);
+    cy.contains('Portal Notifications').should('be.visible');
+  });
+
+  before(() => {
+    cy.log('Import (create) v2 API');
+    cy.request({
+      method: 'POST',
+      url: `${Cypress.env('managementApi')}${Cypress.env('defaultOrgEnv')}/apis/import`,
+      auth: { username: API_PUBLISHER_USER.username, password: API_PUBLISHER_USER.password },
+      body: ApisFaker.apiImport({
+        plans: [PlansFaker.plan({ status: PlanStatus.PUBLISHED })],
+      }),
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      api = response.body;
+    });
+  });
+
+  const ApiDetails = new apiDetails();
+
+  it('Verify notification page elements', () => {
+    cy.getByDataTestId('api_notifications_add_notification').should('be.visible');
+    cy.contains('Portal Notifications').should('be.visible');
+    cy.contains('Default Mail Notifications').should('be.visible');
+  });
+
+  it('Create generic notification', () => {
+    cy.contains('Default Mail Notifications').should('be.visible');
+    cy.getByDataTestId('api_notifications_add_notification').click();
+    cy.contains('New notification', { timeout: 5000 }).should('be.visible');
+    cy.getByDataTestId('api_notifications_create_namefield').type('Test notification');
```

Comment:
It's a good practice to use Faker whenever you need to name items. This way, in case you want to create multiple entries or retry tests you won't face any problems with already existing items with the same name.